scripts/lustre

```diff
@@ -54,15 +54,17 @@ lustre_monitor() {
 
     mntdevs=$(ls /{proc,sys}/fs/lustre/osd-zfs/*/mntdev 2>/dev/null)
     if [ -z "${mntdevs}" ]; then
+        ocf_log info "No mntdev found in proc or sys"
         return $OCF_NOT_RUNNING
     fi
 
     for mntdev_path in $mntdevs ; do
         mntdev=$(cat $mntdev_path 2>/dev/null)
         if [ $mntdev = $OCF_RESKEY_dataset ] ; then
-        ocf_log debug "$OCF_RESKEY_dataset found in $mntdev_path"
-        return $OCF_SUCCESS
+            ocf_log debug "$OCF_RESKEY_dataset found in $mntdev_path"
+            return $OCF_SUCCESS
         fi
+        ocf_log info "$mntdev is not in $OCF_RESKEY_dataset"
```

Comment:
wording is backwards.  Instead how about "$OCF_RESKEY_dataset not found in $mntdev_path"?

Reply:
ping

Reply:
@ofaaland I made the change last week, resolved this thread and requested a new code review.  See this commit: 7ecb78865fb3ebb5fb8b654f9c88ba1a0ac60bd1

Reply:
Oops, sorry I overlooked that.